lib/sequelize.js

```diff
@@ -145,6 +149,7 @@ var Sequelize = function(database, username, password, options) {
     pool: {},
     quoteIdentifiers: true,
     hooks: {},
+    retry: {max: 5, match: ['SQLITE_BUSY: database is locked']},
```

Comment:
I simply meant a sequence of queries inside a transaction :)
But yeah we'll likely need `retry` to only apply to queries and then possible have a global argument like `retryTransaction`